tls/s2n_x509_validator.c

```diff
@@ -64,6 +64,15 @@ uint8_t s2n_x509_ocsp_stapling_supported(void)
     return S2N_OCSP_STAPLING_SUPPORTED;
 }
 
+bool s2n_libcrypto_supports_flag_no_check_time()
```

Comment:
The "s2n_libcrypto" prefix suggests this might not belong here. Maybe https://github.com/aws/s2n-tls/blob/main/crypto/s2n_libcrypto.h instead? Or maybe "s2n_x509_validator_supports_no_check_time_flag". Wow that's a long name.

Reply:
I moved it to s2n_libcrypto.c!